sbt

```diff
@@ -34,11 +34,9 @@ fi
 #http://stackoverflow.com/questions/5374455/what-does-java-option-xmx-stand-for
 jvm_mem="-Xmx$((physical_mem / 2 / 1024))m"
 
-#http://stackoverflow.com/questions/12114174/what-does-xxmaxpermsize-do
-perm_size="-XX:MaxPermSize=$((physical_mem / 3 / 1024))m"
```

Comment:
No longer used in JDK 8
http://stackoverflow.com/questions/18339707/permgen-elimination-in-jdk-8